CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
 
+## 0.14.0 - 2020-07-14
+### Added
+- Use the setting "azureDatabases.batchSize" to configure the Mongo
+"DBQuery.shellBatchSize" attribute and the batch size when loading children in
```

Comment:
How come these bullets are split into separate lines? Seems to render okay in markdown, but makes it a bit difficult to read in raw format

Reply:
No particular reason - just how I usually write markdown. Went ahead & removed the line breaks